### ISSUE TITLE
Passing job options like (queue, run_at) to `JobClass.enqueue` as top level keyword args has been deprecated

### ISSUE THREAD

**Issue by coffenbacher** [hlascelles/que-scheduler#336]
# Description

Our logs are full of deprecation warnings:
```
Passing job options like (queue, run_at) to `JobClass.enqueue` as top level keyword args has been deprecated and will be removed in version 2.0. Please wrap job options in an explicit `job_options` keyword arg instead.
```

# Versions

Please provide the following versions where you see the issue:

ruby version: 2.7.2p137
que version: 1.4.0
que-scheduler version: 4.3.0
postgres version:  14.2

# Messages

The deprecation error doesn't come with a stacktrace or line reference in our logs as far as I can tell...

**Comment by hlascelles** [comment]
Interesting... I think that is coming from que. I have a PR / branch coming to support Que 2.0, but I think it needs to take this into account too. I will have a look!

**Comment by coffenbacher** [comment]
Great, yes, I believe it is a Que warning about how que-scheduler is queueing things. We had to change our own enqueueing format as well to avoid the warnings getting triggered by our project code. It was, fortunately, a small change in our experience.

**Comment by hlascelles** [comment]
OK, I have changed the que 2.0 branch to support this as well. I think it should work for you.

Can you try that branch? If it works out I will add another test and cut a gem!

NB, if you are using the ActiveJob wrapper then you must be on the 2.0.0.beta que gem version to squash that warning message, as they fixed it here: https://github.com/que-rb/que/commit/d9244b9074df17f61fe7564b93bbb6865f49ef20

**Comment by hlascelles** [comment]
I've released 4.3.1 which I believe fixes this: https://rubygems.org/gems/que-scheduler/versions/4.3.1

Let me know if not.
